static/src/javascripts/projects/common/modules/atoms/youtube-tracking.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import mediator from 'lib/mediator';
+import config from 'lib/config';
+import ophan from 'ophan/ng';
+import { buildGoogleAnalyticsEvent } from 'common/modules/video/ga-helper';
+import type { MediaEvent } from 'common/modules/video/ga-helper';
+
+const eventAction = (): string => 'video content';
```

Comment:
does this need to be a function?

Reply:
[buildGoogleAnalyticsEvent](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/video/ga-helper.js#L15) expects a callback, so unfortunately in this case, yes

Reply:
On second thoughts, I can inline it in the call to `buildGoogleAnalyticsEvent`